modules/installation-guide/partials/con_persistent-volume-configuration.adoc

```diff
@@ -5,95 +5,121 @@
+| PV count
+| One per user
+| One per workspace
+| Multiple PVs per workspace
 
-[id="the-common-pvc-strategy_{context}"]
-== The `common` PVC strategy
+| Default
+| yes
+| no
+| no
 
-All workspaces inside a {platforms-namespace} use the same Persistent Volume Claim (PVC) as the default data storage when storing data such as the following in their declared volumes:
 
-* projects
-* workspace logs
-* additional Volumes defined by a use
+| Limitations
+| Maximum one running workspace per user when the PV is in the `ReadWriteOnce` access mode
+| PV count depends on the number of workspaces
+| Unpredictable PV count
 
-When the `common` PVC strategy is in use, user-defined PVCs are ignored, and volumes related to these user-defined PVCs are replaced with a volume that refers to the common PVC.
-In this strategy, all {prod-short} workspaces use the same PVC. When the user runs one workspace, it only binds to one node in the cluster at a time.
+|===
 
-The corresponding containers volume mounts link to a common volume, and sub-paths are prefixed with `_<workspace-ID>_` or `__<original-PVC-name>__`. For more details, see xref:how-subpaths-are-used-in-pvcs_{context}[].
+Persistent volumes (PVs) access mode::
+The nature of the PV determines the available access mode: `ReadWriteMany` or `ReadWriteOnce`.
+See link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[Kubernetes documentation - access mode].
+For example, link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html[Amazon EBS] supports only the `ReadWriteOnce` access mode.
 
-The {prod-short} Volume name is identical to the name of the user-defined PVC. It means that if a machine is configured to use a {prod-short} volume with the same name as the user-defined
-PVC has, they will use the same shared folder in the common PVC.
 
-When a workspace is deleted, a corresponding subdirectory (`$\{ws-id}`) is deleted in the PV directory.
+[id="the-common-storage-strategy_{context}"]
+== The `common` storage strategy
 
-NOTE: The common PVC will be removed when a user's workspaces are all deleted. The PVC will be re-created when a non-ephemeral workspace is started.
+This is the default storage strategy.
+For each user, all workspaces use the same PV for the default data storage.
 
-.Restrictions on using the `common` PVC strategy
+When the user starts a first non-ephemeral workspace, the workspace engine creates a common PV.
 
-When the `common` strategy is used, and a workspace PVC access mode is ReadWriteOnce (RWO), only one node can simultaneously use the PVC.
+When the user starts another non-ephemeral workspace, the workspace engine uses the same common PV.
+The workspace only binds simultaneously to one node in the {orch-name} cluster.
+The workspace engine ignores user-defined volumes. 
```

Comment:
I don't think this is right. In this paragraph you are assuming that a devfile `volume` is a Kubernetes `PersistentVolume`. But that's not accurate. We do not mention that in the devfile specification. The only garantee is that the devfile volume will be translated in a volume (escaping the Copy-on-Write container filesystem) and that it will be persisted after a workspace restart (unless `attributes.persistVolumes` is set to `false`). The kind of volume that will be used by Che is just an implementation detail: saying that "user-defined" volumes are ignoered or that "user-defined volumes are replaced with volumes with subPath" is not right. I would remove these 2 sentences.